support-frontend/assets/pages/contributions-landing/components/ContributionAmount.jsx

```diff
@@ -25,11 +23,11 @@ import SvgPound from 'components/svgs/pound';
 import { selectAmount, updateOtherAmount } from '../contributionsLandingActions';
 import ContributionTextInput from './ContributionTextInput';
 import { ChoiceCardGroup, ChoiceCard } from '@guardian/src-choice-card';
-import type { ChoiceCardsProductSetTestR2Variants } from 'helpers/abTests/abtestDefinitions';
+import type { ChoiceCardsProductSetTestR3Variants } from 'helpers/abTests/abtestDefinitions';
+import { yellowChoiceCard } from './ContributionTypeTabs';
```

Comment:
`yellowChoiceCard` is used by 2 components, so it should really be in a common file, e.g. `choiceCardStyles.jsx`